src/main/webapp/script.js

```diff
@@ -167,9 +191,26 @@ function getUrl() {
       selectedDepth = 20;
     }
   }
-  const url = `/data?depth=${selectedDepth}&mutationNum=${currGraphNum}&nodeName=${nodeName}`;
+  const url = `/data?depth=${selectedDepth}&mutationNum=${currMutationNum}&nodeName=${nodeName}`;
   return url;
 }
+
+/**
+ * Add a list element with the given message to the top of the logs list
+ * @param msg the message to display in the new list element
+ */
+function addToLogs(msg) {
+  const logsList = document.getElementById("log-list");
+  const newMsg = document.createElement("li");
+  newMsg.classList.add("log-msg");
+  newMsg.innerText = msg;
+  logsList.appendChild(newMsg);
+  const allLogs = document.querySelectorAll('.log-msg');
+  for (let i = 0; i < allLogs.length; i++) {
```

Comment:
You only need to remove "error-text" for the very last element, right? Everytime you addToLogs only at most one log (the last one) should be red. 

Also I feel like whenever the user sends a new request to the server, we should set the most recent log message to black as well so that if the new request does not have an error, the logs shouldn't have anything red.

Reply:
think i fixed it! let me know if there are problems with it!